### PR TITLE
Convert module to TypeScript

### PR DESCRIPTION
**Pull request by gzog**
**Description**
- Addresses: #2 
- Use same configuration found in latest version of [expect-ct](https://github.com/helmetjs/expect-ct/tree/21aa018565f96f4a8361c34e374ed4b817d0efcc)

### CODE REVIEW

File: test/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import connect from 'connect';
+import request from 'supertest';
+import { IncomingMessage, ServerResponse } from 'http';
+
+import hidePoweredBy = require('..');
+
```
Comment (EvanHahn) [inline test/index.test.ts:6]: Nit: could you remove this line?

File: package.json [hunk 2]
```diff
@@ -15,25 +15,44 @@
...
     "type": "git",
     "url": "git://github.com/helmetjs/hide-powered-by.git"
   },
   "bugs": "https://github.com/helmetjs/hide-powered-by/issues",
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "files": [
```
Comment (EvanHahn) [inline package.json:29]: Now that you've added this, could you `git rm .npmignore`?

**Review by EvanHahn — CHANGES_REQUESTED** [review]
This is great—thanks so much for doing this!!

A few minor tweaks to make:

1. Could you remove the Standard.js badge from the readme, now that we're not using it?
2. Could you fill in the changelog or just remove it? And could you remove the reference to the changelog from the readme?

Again, this is super helpful—thanks for doing this!

**Comment by gzog** [comment]
@EvanHahn Hey Evan. Thanks for feedback!
Made the following updates based on your comments:
- [x] Removed CHANGELOG.md
- [x] Removed reference to changelog in README.md
- [x] Removed reference to Standard.js badge in README.md
- [x] Removed redundant line in index.test.ts
- [x] Removed .npmignore

**Review by EvanHahn — APPROVED** [review]
Thanks so much for doing this! I'll merge and deploy shortly.

**Comment by EvanHahn** [comment]
This has been released in `hide-powered-by@1.1.0`! This was a big help, thanks!!

Please add yourself to the [contributors list](https://github.com/helmetjs/helmetjs.github.io/blob/master/contributors/index.md) when you have a moment.

**Pull request merged**